apps/condo/domains/common/utils/codegeneration/generate.server.utils.js

```diff
@@ -52,6 +53,10 @@ function generateServerUtils (gql) {
         })
     }
 
+    async function getFirst (context, where, params = {}) {
+        return await getAll(context, where, { first: 1, ...params })
+    }
+
```

Comment:
I don't think the getFirst will help a lot. It's easier to use `getAll( ..., ..., { first: 1})` directly.

But, something like `getOne( ... )`:

```
    async function getOne (context, where, params = {}) {
        const objs = await getAll(context, where, { first: 2, ...params })
        if (objs.length > 1) throw new Error('getOne() return more than one result')
        else if (objs.length === 1) return objs[0]
        return undefined
    }

```

can help us to avoid errors in logic

Reply:
done